01_auth_api/src/db/index.types.ts

```diff
@@ -0,0 +1 @@
+export type QueryValues = Array<string | number | boolean | null | Date>;
```

Comment:
A better way to make QueryValue a single type:

`export type QueryValue = string | number | boolean | null | Date;` -- you make this reusable if code starts scaling
and after that make this:
`export type QueryValues = Array<QueryValue>;`

Reply:
Gotcha!

Reply:
Good, thanks